zcash_client_sqlite/src/wallet.rs

```diff
@@ -2350,10 +2376,11 @@ pub(crate) fn put_tx_meta(
 ) -> Result<i64, SqliteClientError> {
     // It isn't there, so insert our transaction into the database.
     let mut stmt_upsert_tx_meta = conn.prepare_cached(
-        "INSERT INTO transactions (txid, block, tx_index)
-        VALUES (:txid, :block, :tx_index)
+        "INSERT INTO transactions (txid, block, mined_height, tx_index)
+        VALUES (:txid, :block, :block, :tx_index)
         ON CONFLICT (txid) DO UPDATE
         SET block = :block,
+            mined_height = :mined_height,
```

Comment:
```suggestion
            mined_height = :block,
```
There is no `:mined_height` named param.